src/Elastic/QueryEngine/DescriptionInterpreters/ValueDescriptionInterpreter.php

```diff
@@ -164,7 +164,7 @@ private function proximity_bool( $field, $value ) {
 
 		// Wide proximity uses ~~ as identifier as in [[~~ ... ]] or
 		// [[in:fox jumps]]
-		if ( $value[0] === '~' ) {
+		if ( is_array( $value ) && $value[0] === '~' ) {
```

Comment:
I'm not entirely sure if this is the correct fix?

doing:

```
$test = '*';

var_dump(strpos( $test, '*' ));

```

throws this:

```
Fatal error: Uncaught TypeError: strpos(): Argument #1 ($haystack) must be of type string, array given
```

unless this should be `$value === '~'`?

Reply:
oh

```
$test = '*';

var_dump($test[0]);
```

works.

Reply:
Think I may have found a better fix.